shell/browser/api/electron_api_app_mac.mm

```diff
@@ -58,6 +58,15 @@
   [NSApp setActivationPolicy:activation_policy];
 }
 
+bool App::IsRunningUnderRosettaTranslation() {
+  int ret = 0;
```

Comment:
_(nit)_ unlike the sample Apple code, this function does not return `ret` so the variable name is weird here. Maybe rename to "current_value" or "cur"?